src/components/MoneyRequestConfirmationList.tsx

```diff
@@ -709,14 +757,28 @@ function MoneyRequestConfirmationList({
                     <FormHelpMessage
                         style={[styles.ph1, styles.mb2]}
                         isError
-                        message={formError}
+                        message={!shouldShowReadOnlySplits ? debouncedFormError : formError}
```

Comment:
Hi ✋ Coming from https://github.com/Expensify/App/issues/43184

The check `shouldShowReadOnlySplits` should be used on the split request, so adding `isTypeSplit` to avoid the `debouncedFormError` called on the non-split request.